core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -371,13 +371,22 @@ trait WhiskWebActionsApi extends Directives with ValidateRequestSize with PostAc
     `Access-Control-Allow-Methods`(OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH),
     `Access-Control-Allow-Headers`(`Authorization`.name, `Content-Type`.name))
 
+  private def contentTypeFromEntity(entity: HttpEntity) = {
+    val noContentType = ContentType(MediaType.customBinary("none", "none", MediaType.NotCompressible))
+
+    entity.contentType match {
+      case ct if ct == noContentType => Seq()
```

Comment:
scala tip, this could be None and Some, which you can add to a seq because they're iterable.